uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/documents.js

```diff
@@ -43,25 +43,21 @@ module.exports = page.extend({
     await this.puppet.click(documentSelector + ' .dropdown-menu a[ng-click*="modalDocument"]');
 
     return this.waitForModal('document');
-  },
+  }
 
   /**
    * Shows the "select dates" filter
    */
   async selectDates () {
     await this.puppet.click('.ct-select-dates');
     await this.puppet.waitFor(500);
-  },
+  }
 
   /**
    * Waits until the user name in the "Staff" column and the filter dates are visible
    */
   async waitForReady () {
     await this.puppet.waitFor('.ct-filter-date', { visible: true });
-    await this.puppet.waitFor('.ct-table-documents [href^="/civicrm/contact/view"]', { visible: true });
-    // For some reason Puppetteer considers the user name visible even when it
-    // isn't really yet, this slight delay allows the element to be fully visible
-    // before taking the screenshots
-    await this.puppet.waitFor(500);
+    await this.puppet.waitFor(4000);
```

Comment:
I know the explanation is mentioned in the description, but still, any reason why it doesn't show it in 500ms? Is there an animation or is it just Backstop instability?

Reply:
no, civihr's performance unfortunately. and it gets worse as we increase the number of screenshots captured in parallel